matRadParetoGUI.m

```diff
@@ -0,0 +1,47 @@
+function hGUI = matRadParetoGUI(varargin)
+% matRad compatability function to call the matRad_MainGUI
```

Comment:
again wrong documentation

Reply:
Is this function actually needed and not only the class call could be used? 
I mainly did this for the matRad Main GUI to behave like before / downwards compatibility, not for a general design choice.

Reply:
Not needed no, I mainly went along the lines of the base GUI